WordPress/src/main/java/org/wordpress/android/ui/comments/NotificationCommentDetailFragment.kt

```diff
@@ -26,6 +30,27 @@ class NotificationCommentDetailFragment : CommentDetailFragment() {
         }
     }
 
+    override fun getUserProfileUiState(): BottomSheetUiState.UserProfileUiState {
+        val user = mContentMapper.mapToFormattableContentList(mNote!!.body.toString())
+            .find { FormattableRangeType.fromString(it.type) == FormattableRangeType.USER }
+
+
+        return BottomSheetUiState.UserProfileUiState(
+            userAvatarUrl = mNote!!.iconURL,
```

Comment:
I'm not sure if it's safe to use the not-null assertion for `mNote`. Shouldn't we use the same approach as for the `mComment`?

Reply:
In `NotificationCommentDetailFragment`, `mNote` should be non-null.